examples/simplesamlphp/docker-compose.yml

```diff
@@ -13,10 +13,10 @@ php:
 web:
   image: nginx:alpine
   ports:
-    - "8080:80"
+    - "8081:80"
```

Comment:
I take this to mean you had something else already running on :8080 ... is this change necessary?